boards/xtensa/intel_adsp_cavs15/doc/index.rst

```diff
@@ -126,7 +126,7 @@ a path to the same key file used above.
 
     ./scripts/twister --device-testing -p intel_adsp_cavs15 \
       --device-serial-pty $ZEPHYR_BASE/soc/xtensa/intel_adsp/tools/cavstool_client.py,cavs15,-l \
-      --west-flash "--remote-host=cavs15,--pty"
+      --west-flash "--remote-host=cavs15"
```

Comment:
```suggestion
      --west-flash "--remote-host=myboard.local"
```
Please replace `cavs15` as a sample hostname, it looks exactly like a board name and SoC series name.